firmware/fx2/descriptors_hdmi2usb.a51

```diff
@@ -487,6 +495,30 @@ string2:
 	.db    'U', 00
 	.db    'S', 00
 	.db    'B', 00
+	.db    '.', 00
+	.db    't', 00
+	.db    'v', 00
+	.db    ' ', 00
+	.db    '-', 00
+	.db    ' ', 00
+	.db    'N', 00
```

Comment:
Need to figure out a good way to customize this based on the `BOARD=xxx` environment variable.